pkg/backuphelpers/backupvars.go

```diff
@@ -0,0 +1,53 @@
+package backuphelpers
```

Comment:
```suggestion
package backuphelpers

package backuphelpers

import (
	"github.com/openshift/api/config/v1alpha1"
	"sync"
)

type BackupVar struct {
	m         sync.Mutex
	enabled   bool
	backupCrd v1alpha1.EtcdBackupSpec
}

```

would be plenty enough, with a Setter and a Getter.

Reply:
and a function that returns this as an argument list:

```
func (v BackupVar) ArgString() string {
...
}

```

Reply:
done 👍🏽